perun-openapi/openapi.yml

```diff
@@ -5613,122 +5848,797 @@ paths:
       summary: |
         Log arbitrary auditer message/event to the audit log.
       parameters:
-        -  { name: msg, in: query, schema: { type: string }, description: "Message to be logged", required: true }
+        - { name: msg, in: query, schema: { type: string }, description: "Message to be logged", required: true }
       responses:
         '200':
           $ref: '#/components/responses/VoidResponse'
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-
-
   #################################################
   #                                               #
-  # DatabaseManager                               #
+  # CabinetManager                                #
   #                                               #
   #################################################
 
-  /json/databaseManager/getCurrentDatabaseVersion:
+  /json/cabinetManager/getPublicationSystems:
     get:
       tags:
-        - DatabaseManager
-      operationId: getCurrentDatabaseVersion
-      summary: Returns current database version, e.g. "3.1.55"
+        - CabinetManager
+      operationId: getPublicationSystems
+      summary: Get all PublicationSystems in Perun. If none, return empty list.
       responses:
         '200':
-          $ref: '#/components/responses/StringResponse'
+          $ref: '#/components/responses/ListOfPublicationSystemsResponse'
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  /json/databaseManager/getDatabaseDriverInformation:
-    get:
+  /urlinjsonout/cabinetManager/deletePublicationSystem:
+    post:
       tags:
-        - DatabaseManager
-      operationId: getDatabaseDriverInformation
-      summary: Returns current database driver info, e.g. "PostgreSQL JDBC Driver-42.2.8"
+        - CabinetManager
+      operationId: deletePublicationSystem
+      summary: Deletes PublicationSystem by its ID.
+      parameters:
+        - $ref: '#/components/parameters/id'
       responses:
         '200':
-          $ref: '#/components/responses/StringResponse'
+          $ref: '#/components/responses/VoidResponse'
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  /json/databaseManager/getDatabaseInformation:
-    get:
+  /urlinjsonout/cabinetManager/createPublicationSystem:
+    post:
       tags:
-        - DatabaseManager
-      operationId: getDatabaseInformation
-      summary: Returns current database info, e.g. "PostgreSQL-12.0 (Debian 12.0-2.pgdg100+1)"
+        - CabinetManager
+      operationId: createPublicationSystem
+      summary: Creates new PublicationSystem.
       responses:
         '200':
-          $ref: '#/components/responses/StringResponse'
+          $ref: '#/components/responses/PublicationSystemResponse'
         default:
           $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
```

Comment:
I think this would not work. When you call the `urlinjsonout` serializer/deserialzer, the RPC expects you to put the data in URL parameters, not in the request body. See other methods in the specification. Or you can change the serializer to `json`.

The general rule is that you call `urlinjsonout` when you are calling POST and you are passing only ids of objects. If you call POST and want to pass some object, then you call the `json` serializer and pass the data in the request body.

If you have any questions you can write to me on Slack.